js/views/homeVw.js

```diff
@@ -13,7 +15,7 @@ var loadTemplate = require('../utils/loadTemplate'),
     userShortModel = require('../models/userShortMd'),
     simpleMessageView = require('./simpleMessageVw');
 
-module.exports = Backbone.View.extend({
+module.exports = baseVw.extend({
```

Comment:
In order to take advantage of the functionality in the base view, your views needs to extend from it. For all practical purposes, all our views should do this. If we find other functionality common to all views, we could consider adding it to the base view.